src/Microsoft.DotNet.Build.Tasks.VersionTools/Microsoft.DotNet.Build.Tasks.VersionTools.csproj

```diff
@@ -0,0 +1,44 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <OutputType>Library</OutputType>
+    <RootNamespace>Microsoft.DotNet.Build.Tasks.VersionTools</RootNamespace>
+    <AssemblyName>Microsoft.DotNet.Build.Tasks.VersionTools</AssemblyName>
+    <CLSCompliant>false</CLSCompliant>
+    <AssemblyVersion>1.0.0.0</AssemblyVersion>
+    <ProjectGuid>{D4D6ACB6-9D73-488E-B184-4A1C00D026D2}</ProjectGuid>
+    <NuGetTargetMoniker>.NETStandard,Version=v1.5</NuGetTargetMoniker>
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
+  </PropertyGroup>
+  <PropertyGroup>
+    <StartupObject />
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Include="..\Microsoft.DotNet.Build.Tasks\BuildTask.cs" />
```

Comment:
Looks that way. We should avoid that if possible.

Reply:
I wasn't certain that I could convince myself someone didn't take a dependency on the task in BuildTools, so it seems safer to duplicate this in the assemblies, but share the code.

Reply:
I followed the versioning of the other packages produced by BuildTools.  Are you proposing that all of these get updated?

```
Microsoft.DotNet.Build.CloudTest.1.0.0-prerelease-02027-0.nupkg
Microsoft.DotNet.Build.Tasks.Feed.1.0.0-prerelease-02027-0.nupkg
Microsoft.DotNet.Build.Tasks.VersionTools.1.0.0-prerelease-02027-0.nupkg
Microsoft.DotNet.BuildTools.2.0.0-prerelease-02027-0.nupkg
Microsoft.DotNet.BuildTools.ApiCompat.1.0.0-beta3-02027-0.nupkg
Microsoft.DotNet.BuildTools.Cci.1.0.0-prerelease-02027-0.nupkg
Microsoft.DotNet.BuildTools.GenAPI.1.0.0-beta3-02027-0.nupkg
Microsoft.DotNet.BuildTools.GenFacades.1.0.0-beta3-02027-0.nupkg
Microsoft.DotNet.BuildTools.RepoUtils.1.0.1-prerelease-02027-0.nupkg
Microsoft.DotNet.BuildTools.Run.1.0.1-prerelease-02027-0.nupkg
Microsoft.DotNet.BuildTools.TestSuite.1.0.1-prerelease-02027-0.nupkg
Microsoft.DotNet.BuildTools.VstsBuildsApi.1.0.27-prerelease-02027-0.nupkg
Microsoft.DotNet.VersionTools.1.0.27-prerelease-02027-0.nupkg
Microsoft.xunit.extensibility.execution.netcore.1.0.1-prerelease-02027-0.nupkg
Microsoft.xunit.netcore.extensions.1.0.1-prerelease-02027-0.nupkg
xunit.console.netcore.1.0.4-prerelease-02027-0.nupkg
```

Reply:
I wonder if instead of this we just package the task library in its own package for folks that want to consume it directly. That will mean things stay in the same place and I don't think it is a big deal to have more stuff in the task library as it won't add much in size and it will move us closer to componentizing buildtools. I think that is better then one-off creating a new task library for one small task.